pom.xml

```diff
@@ -1,354 +1,354 @@
 <project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
what did you do here? please reset this file

Reply:
`git checkout upstream/master pom.xml`

Reply:
I just updated the Spring Boot version, nothing else....